independent-projects/qute/core/src/main/java/io/quarkus/qute/Template.java

```diff
@@ -44,6 +42,69 @@ default TemplateInstance data(String key, Object data) {
         return instance().data(key, data);
     }
 
+    /**
+     * 
+     * @param key1
+     * @param data1
+     * @param key2
+     * @param data2
+     * @return a new template instance
+     */
+    default TemplateInstance data(String key1, Object data1, String key2, Object data2) {
```

Comment:
What's the motivation for that? I personally find more legible to do `data(key1,data1).data(key2,data2)` in my code instead

Reply:
Well, that's a matter of taste. I think that it's useful for simple cases where just a few key/value pairs is used. And I'm probably not the only one. See for example JDK9+ `Map.of()` or `com.google.common.collect.ImmutableMap.of()` methods.

Reply:
I think that other impls are there because you can't chain Map.put statements, but I approved it anyway ;)